src/aiida/cmdline/commands/cmd_computer.py

```diff
@@ -766,10 +766,10 @@ def computer_export_setup(computer, output_file, overwrite, sort):
         'append_text': computer.get_append_text(),
     }
 
+    if output_file is None:
+        output_file = pathlib.Path(f'{computer.label}-setup.yml')
```

Comment:
here you use `.yml` but in code export the default is `.yaml`. Best to be consistent. I don't really care but if we have to chose, the YAML spec suggests to use .yaml

Reply:
Ah, yes, I just overlooked that. Thanks! Yeah, I mean the file format is called `YAML` not `YML`, so I don't see any reason why one should use the latter.